scanapi/scan_api.go

```diff
@@ -47,8 +47,8 @@ type Filesystem interface {
 
 // SQLQuerier is an interface that supports SQL queries to a target SQL database.
 type SQLQuerier interface {
-	// SQLQuery executes SQL queries to a target SQL database and returns the number of result rows.
-	SQLQuery(ctx context.Context, query string) (int, error)
+	// SQLQuery executes SQL queries to a target SQL database and returns the number of result rows and the result as string.
```

Comment:
We can infer the number of result rows from the result string, right? In that case let's remove the result row count from the return values.

Reply:
Yes we can, but this removal would have an impact on other functionalities than the one we are integrating. 
The number of result rows was intentionally left in order to have the lowest number of modification in the current codebase.
Given this information, would you suggest to remove it anyway, adapting the current functionalities which are using it?

Reply:
Ah sorry, missed this question. I think it's fine to remove the result row count regardless. Users of the binary won't see a difference and users of Localtoast as a library currently don't use the SQL feature, internally at Google least. So I'm not aware of any user's workflow breaking because of this change.

Reply:
@erikvarga I'm asking you a double check, just to be sure: as per your request [here](https://github.com/google/localtoast/pull/23/#discussion_r1551615038) we are storing only the first row in the result string. Due to this, we are not able to infer the number of rows in other points of the code than in sql_querier.go.
From the functionality point of view, it should be okay since localtoast only cares if the query has results or not, so this modification should be feasible with no impact on the user experience.

If you agree with that, i'll proceed to push this modifications too in the current PR.

Reply:
Hi @erikvarga, do you have any feedback on this?

Reply:
Ah sorry I did not see this comment. Yes, the number of rows is currently only used to check if there are any results returned or not, which can be inferred from just the first row's content too. So it should be fine to add this change to the PR.

Reply:
No worries! Done and pushed.